great_expectations/data_context/store/datasource_store.py

```diff
@@ -64,14 +67,15 @@ def remove_key(self, key: Union[DataContextVariableKey, GeCloudIdentifier]) -> N
         """
         return self._store_backend.remove_key(key.to_tuple())
 
-    def serialize(self, value: DatasourceConfig) -> Union[str, DatasourceConfig]:
+    def serialize(self, value: DatasourceConfig) -> Union[str, dict, DatasourceConfig]:
         """
         See parent 'Store.serialize()' for more information
         """
         if self.ge_cloud_mode:
             # GeCloudStoreBackend expects a json str
             return self._schema.dump(value)
```

Comment:
Can we remove this fork now?

Reply:
Ooh yeah I missed this one - I think so.